mailet/pom.xml

```diff
@@ -51,6 +51,7 @@
         <module>crypto</module>
         <module>icalendar</module>
         <module>mailetdocs-maven-plugin</module>
+        <module>scala</module>
```

Comment:
Why don't we add this matcher in standard package?

We can very well combine java & scala in the same mvn module...

Reply:
We can, but I don't really see any good reason to do so. Do you have some?

Reply:
Increasing maven module count makes release process harder.

So I think it should be the other way around: we don't create yet another maven module unless we have a good reason to do so.

Reply:
good reason: not pulling scala stdlib in James for a single mailet that most people won't use.

Reply:
scala std lib should already be there anyway: see task-api dependency. Non goal.

Reply:
Or shall we remove scala usage from task-api too?

Reply:
> scala std lib should already be there anyway: see task-api dependency. Non goal.

True.

> Or shall we remove scala usage from task-api too?

is it a real question?